src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -10,79 +10,72 @@
-    --pf-c-about-modal-box--PaddingBottom: var(--pf-global--spacer--3xl);
-    --pf-c-about-modal-box--PaddingLeft: var(--pf-global--spacer--3xl);
+    --pf-c-about-modal-box--Padding: var(--pf-c-about-modal-box--sm--Padding);
   }
 
   // Element variables
 
   // Brand
   // Set default padding top, right, left to shared values
   --pf-c-about-modal-box__brand--PaddingTop: var(--pf-global--spacer--2xl);
-  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
-  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
-  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--Padding);
+  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box--Padding);
+  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box--Padding);
 
   // Close
   // Set default padding top, right, bottom to shared values
   --pf-c-about-modal-box__close--PaddingTop: var(--pf-c-about-modal-box__brand--PaddingTop);
-  --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box--Padding);
   --pf-c-about-modal-box__close--PaddingBottom: var(--pf-c-about-modal-box__brand--PaddingBottom);
+  --pf-c-about-modal-box__close--sm--PaddingRight: var(--pf-c-about-modal-box--Padding);
+  --pf-c-about-modal-box__close--lg--PaddingRight: var(--pf-c-about-modal-box--Padding);
   --pf-c-about-modal-box__close--ZIndex: var(--pf-global--ZIndex--2xl);
 
   // Brand img
   --pf-c-about-modal-box__brand-image--Height: #{pf-size-prem(40px)};
 
   // Header
   // Set default padding right, bottom, left to shared values
-  --pf-c-about-modal-box__header--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
+  --pf-c-about-modal-box__header--PaddingRight: var(--pf-c-about-modal-box--Padding);
   --pf-c-about-modal-box__header--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-about-modal-box__header--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__header--PaddingLeft: var(--pf-c-about-modal-box--Padding);
 
   // Strapline
   --pf-c-about-modal-box__header-strapline--PaddingBottom: var(--pf-global--spacer--md);
 
   // Content
   // Using global padding values for margins to prevent scrollbar extending past content
   // Set default margin top, right, bottom, left to shared values
-  --pf-c-about-modal-box__content--MarginTop: var(--pf-c-about-modal-box--PaddingTop);
-  --pf-c-about-modal-box__content--MarginRight: var(--pf-c-about-modal-box--PaddingRight);
-  --pf-c-about-modal-box__content--MarginBottom: var(--pf-c-about-modal-box--PaddingBottom);
-  --pf-c-about-modal-box__content--MarginLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__content--Margin: var(--pf-c-about-modal-box--Padding);
```

Comment:
Is using Padding for Margin an exception ? Or can you give --pf-c-about-modal-box__content--Margin its own value?